`TestParentDeletion` fails on OpenBSD

This test is failing non-deterministically:
```
fremen% STRACE=model go test -v -run TestParentDeletion                                                                            
=== RUN   TestParentDeletion
2018/10/07 13:09:02.267277 set.go:80: INFO: No stored folder metadata for "default": recalculating                                 
2018/10/07 13:09:02.268209 model.go:197: INFO: Ready to synchronize "default" (default) (sendreceive)                              
2018/10/07 13:09:02.268334 model.go:1640: INFO: Device GYRZZQB-IRNPV4Z-T7TC52W-EQYJ3TT-FDQW6MW-DFLMU42-SSSU6EM-FBK2VAY client is " " named "" at
2018/10/07 13:09:02.287054 folder.go:290: INFO: Completed initial scan of sendreceive folder "default" (default)                   
--- PASS: TestParentDeletion (0.58s)
PASS
ok      github.com/syncthing/syncthing/lib/model        0.751s
fremen% STRACE=model go test -v -run TestParentDeletion
=== RUN   TestParentDeletion
2018/10/07 13:09:05.467361 set.go:80: INFO: No stored folder metadata for "default": recalculating                                 
2018/10/07 13:09:05.468368 model.go:197: INFO: Ready to synchronize "default" (default) (sendreceive)                              
2018/10/07 13:09:05.468500 model.go:1640: INFO: Device GYRZZQB-IRNPV4Z-T7TC52W-EQYJ3TT-FDQW6MW-DFLMU42-SSSU6EM-FBK2VAY client is " " named "" at
2018/10/07 13:09:05.488119 folder.go:290: INFO: Completed initial scan of sendreceive folder "default" (default)                   
--- PASS: TestParentDeletion (0.60s)
PASS
ok      github.com/syncthing/syncthing/lib/model        0.748s
fremen% STRACE=model go test -v -run TestParentDeletion
=== RUN   TestParentDeletion
2018/10/07 13:09:08.717217 set.go:80: INFO: No stored folder metadata for "default": recalculating
2018/10/07 13:09:08.718186 model.go:197: INFO: Ready to synchronize "default" (default) (sendreceive)
2018/10/07 13:09:08.718315 model.go:1640: INFO: Device GYRZZQB-IRNPV4Z-T7TC52W-EQYJ3TT-FDQW6MW-DFLMU42-SSSU6EM-FBK2VAY client is " " named "" at 
2018/10/07 13:09:08.737205 folder.go:290: INFO: Completed initial scan of sendreceive folder "default" (default)
--- FAIL: TestParentDeletion (0.09s)
    requests_test.go:589: remove /home/porter/go/src/github.com/syncthing/syncthing/lib/model/testdata/_request-402399101/foo: directory not empty
FAIL
```

Version: `f528923d1ef224a07d58fea21b0dca14ca18ba78`.

Thanks

Hi,

I'm able to reproduce this one on my laptop. It seems sensitive to the machine we are running on. I remember not being able to repro on my desktop.

I suppose this is the same inotify issue @AudriusButkevicius ? If so, can we skip this test also?

```
--- FAIL: TestParentDeletion (0.02s)
    requests_test.go:606: remove /tmp/_request-182950755/foo: directory not empty
FAIL
FAIL    github.com/syncthing/syncthing/lib/model        58.983s
```

^ on master.

Thanks

Thanks imsodin.

I'm not sure how insightful this will be, but I ran the test in a loop until it failed. It's non-deterministic.

```
$ for i in {0..50}; do env STRACE="model fs" go test -v -run TestParentDeletion; done
...
=== RUN   TestParentDeletion
15:12:14 INFO: No stored folder metadata for "default": recalculating
15:12:14 INFO: Ready to synchronize "default" (default) (sendreceive)
15:12:14 INFO: Device GYRZZQB-IRNPV4Z-T7TC52W-EQYJ3TT-FDQW6MW-DFLMU42-SSSU6EM-FBK2VAY client is " " named "" at                                                                                                    
15:12:14 INFO: Completed initial scan of sendreceive folder "default" (default)
--- PASS: TestParentDeletion (0.59s)
PASS
ok      github.com/syncthing/syncthing/lib/model        0.724s
```

vs. 

```
=== RUN   TestParentDeletion
15:12:16 INFO: No stored folder metadata for "default": recalculating
15:12:16 INFO: Ready to synchronize "default" (default) (sendreceive)
15:12:16 INFO: Device GYRZZQB-IRNPV4Z-T7TC52W-EQYJ3TT-FDQW6MW-DFLMU42-SSSU6EM-FBK2VAY client is " " named "" at                                                                                                    
15:12:16 INFO: Completed initial scan of sendreceive folder "default" (default)
--- FAIL: TestParentDeletion (0.07s)
    requests_test.go:606: remove /tmp/_request-879856505/foo: directory not empty
FAIL
```

--- a/lib/model/model_test.go
+++ b/lib/model/model_test.go
@@ -50,6 +50,7 @@ func init() {
 
 	defaultFolderConfig = config.NewFolderConfiguration(protocol.LocalDeviceID, "default", "default", fs.FilesystemTypeBasic, "testdata")
 	defaultFolderConfig.Devices = []config.FolderDeviceConfiguration{{DeviceID: device1}}
+	defaultFolderConfig.FSWatcherEnabled = false
 	defaultCfg = config.Configuration{
 		Version: config.CurrentVersion,
 		Folders: []config.FolderConfiguration{defaultFolderConfig},

--- a/lib/model/requests_test.go
+++ b/lib/model/requests_test.go
@@ -718,6 +718,7 @@ func setupModelWithConnection() (*Model, *fakeConnection, string, *config.Wrappe
 	cfg := defaultCfgWrapper.RawCopy()
 	cfg.Devices = append(cfg.Devices, config.NewDeviceConfiguration(device2, "device2"))
 	cfg.Folders[0] = config.NewFolderConfiguration(protocol.LocalDeviceID, "default", "default", fs.FilesystemTypeBasic, tmpDir)
+	cfg.Folders[0].FSWatcherEnabled = false
 	cfg.Folders[0].Devices = []config.FolderDeviceConfiguration{
 		{DeviceID: device1},
 		{DeviceID: device2},

